integration/testdata/plugin/local/jib/skaffold.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: skaffold/v1beta7
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/sk8-skaffold/skaffold-jib
+    context: .
```

Comment:
@priyawadhwa I noticed that if I don't give `context: .`, Jib fails [here](https://github.com/GoogleContainerTools/skaffold/blob/v0.24.0/pkg/skaffold/jib/jib.go#L36) because `cmd.Dir` (which is `artifact.Workspace`) is an empty string. I noticed `context: .` is also given for the [Bazel plugin example project](https://github.com/GoogleContainerTools/skaffold/blob/v0.24.0/integration/examples/test-plugin/local/bazel/skaffold.yaml#L6). Is this required? Shouldn't someone set `artifact.Workspace` if this is missing?

Reply:
Hey @chanseokoh, so there's a [setDefaultWorkspace](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/schema/defaults/defaults.go#L186) function but it looks like it [doesn't get set](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/schema/defaults/defaults.go#L65) when using plugins.

This looks like a bug I introduced, if you move it out to before the plugins [check](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/schema/defaults/defaults.go#L62) this should work as expected (and thanks for catching the bug!)